pkg/ctl/functions/upload_test.go

```
@@ -128,6 +125,8 @@ func TestUploadAndDownloadCommands(t *testing.T) {
 		log.Fatal(err)
 		t.Fail()
 	}
+	defer os.RemoveAll(f.Name())
```

Old behavior looks better.

---

use `log.Panic` instead of `log.Fatal`, since the `log.Fatal` will skip `defer` calls.

---

Good catch!